src/visGeometry/VolumeModel.ts

```diff
@@ -1,40 +1,136 @@
-import { Box3Helper, Euler, Object3D, Vector3 } from "three";
+import { Euler, Object3D, Vector3 } from "three";
 
-import { Volume, VolumeDrawable } from "@aics/volume-viewer";
+import {
+    HasThreeJsContext,
+    Lut,
+    Volume,
+    VolumeDrawable,
+} from "@aics/volume-viewer";
 
 import { AgentData } from "../simularium/types";
 
-// TEMPORARY HACK to make things typecheck. TODO remove!
-import { VolumeRenderImpl } from "@aics/volume-viewer/es/types/VolumeRenderImpl";
-interface TempRayMarchedVolume extends VolumeRenderImpl {
-    boxHelper: Box3Helper;
-}
-
 export default class VolumeModel {
     // TODO what to do with this `cancelled` property? Type check fails without it.
     // When should it be set, if ever; what should it be used for, if anything?
     public cancelled = false;
-    private image?: VolumeDrawable;
+    private drawable?: VolumeDrawable;
```

Comment:
If I recall, I think the "cancelled" property mentioned above is for when the trajectory has been changed but there is still geometry download requests in-flight.  We can use `cancelled` to just drop the arriving stuff.